examples/oft-solana/tasks/common/wire.ts

```diff
@@ -165,7 +164,7 @@ task(TASK_LZ_OWNABLE_TRANSFER_OWNERSHIP)
     //
     // Only pass this if you deployed a new OFT program, if you are using the default
     // LayerZero OFT program you can omit this
-    .addParam('solanaProgramId', 'The OFT program ID to use', OftProgram.OFT_DEFAULT_PROGRAM_ID, publicKey, true)
+    .addParam('solanaProgramId', 'The OFT program ID to use', undefined, publicKey, true)
```

Comment:
Should we remove this task argument altogether?

Reply:
We actually need to require the user provides a `solanaProgramId`, but can't add it as a required param.  The limitation of the task overriding is that the newly added params cannot be added as required.  #893 shows this failing:

https://github.com/LayerZero-Labs/devtools/actions/runs/10926300748/job/30329833041?pr=893#step:8:1095